python/cugraph-pyg/cugraph_pyg/tests/mg/test_mg_cugraph_store.py

```diff
@@ -220,283 +158,292 @@ def test_edge_attr():
-                base_series = base_series.compute().to_cupy()
-
-                tsr = feature_store.multi_get_tensor(
-                    [
-                        [
-                            vertex_type,
-                            property_name,
-                            vertex_ids,
-                            [property_name],
-                            cupy.int64,
-                        ]
-                    ]
-                )
-                assert len(tsr) == 1
-                tsr = tsr[0]
+        tsr = cugraph_store.multi_get_tensor(
+            [
+                CuGraphTensorAttr(vertex_type, feat_name, v_ids)
+                for feat_name in feat_names
+            ]
+        )
 
-                assert list(tsr) == list(base_series)
+        assert np.stack(tsr).get().tolist() == base_series.tolist()
 
 
-def test_get_all_tensor_attrs(graph):
-    pG = graph
-    feature_store, graph_store = to_pyg(pG, backend="cupy")
+def test_get_all_tensor_attrs(graph, dask_client):
+    F, G, N = graph
+    cugraph_store = CuGraphStore(F, G, N, backend="cupy", multi_gpu=True)
 
     tensor_attrs = []
-    for vertex_type in pG.vertex_types:
-        tensor_attrs.append(
-            CuGraphTensorAttr(
-                vertex_type, "x", properties=["prop1", "prop2"], dtype=cupy.float32
+    for vertex_type in sorted(N.keys()):
+        for prop in ["prop1", "prop2"]:
+            tensor_attrs.append(
+                CuGraphTensorAttr(
+                    vertex_type,
+                    prop,
+                    properties=None,
+                    dtype=F.get_data([0], vertex_type, "prop1").dtype,
+                )
             )
-        )
 
-    assert tensor_attrs == feature_store.get_all_tensor_attrs()
+    for t in tensor_attrs:
+        print(t)
 
+    print("\n\n")
 
-def test_get_tensor_size(graph):
-    pG = graph
-    feature_store, graph_store = to_pyg(pG, backend="cupy")
+    for t in cugraph_store.get_all_tensor_attrs():
+        print(t)
 
-    vertex_types = pG.vertex_types
-    for vertex_type in vertex_types:
-        for property_name in pG.vertex_property_names:
-            if property_name != "vertex_type":
-                base_series = pG.get_vertex_data(
-                    types=[vertex_type], columns=[property_name]
-                )
+    assert sorted(tensor_attrs, key=lambda a: (a.group_name, a.attr_name)) == sorted(
+        cugraph_store.get_all_tensor_attrs(), key=lambda a: (a.group_name, a.attr_name)
+    )
 
-                vertex_ids = base_series[pG.vertex_col_name]
-                vertex_ids = vertex_ids.compute().to_cupy()
-                size = feature_store.get_tensor_size(
-                    vertex_type, property_name, vertex_ids, [property_name], cupy.int64
-                )
 
-                assert len(base_series) == size
+@pytest.mark.skip("not implemented")
+def test_get_tensor_spec_props(graph, dask_client):
+    pass
```

Comment:
It might be better to replace `pass` with `raise NotImplementedError` simply so if someone tells pytest to run skipped tests, this won't pass making them think it works.

Reply:
resolved